create.go

```diff
@@ -140,6 +140,7 @@ func getSchemaForType(name string, item *cty.Type) *openapi_spec.Schema {
 
 		// Handle more complex types - map, set and list
 	} else if item.IsMapType() {
+		// Question: Is this not an infinite loop?
```

Comment:
It's a recursive loop that will run until no more maps or nested types are found so not infinite, unless I've messed something up!

Reply:
Aaa, `item.MapElementType()` resolves into a string or something other than a map, I think @jjcollinge

Reply:
Yeh looks ok to me

Reply:
np, Can we remove the comment?